datalad/support/annexrepo.py

```diff
@@ -120,26 +181,150 @@ def annex_get(self, files, **kwargs):
+        # TODO: By now git_cmd is expected to be string. Figure out how to deal with a list here.
+
+        if not self.is_direct_mode():
+            lgr.warning("annex_proxy() called in indirect mode: %s" % git_cmd)
+            raise CommandNotAvailableError(cmd=cmd_str, msg="Proxy doesn't make sense if not in direct mode.")
+        try:
+            output = self.cmd_call_wrapper(cmd_str, shell=True, return_output=True, cwd=self.path)
+        except CommandError, e:
+            if "Unknown command" in output[1]:
+                raise CommandNotAvailableError(cmd=cmd_str, msg=e.msg, stderr=e.stderr, stdout=e.stdout)
+            else:
+                raise e
+        return output[0]
+
+    @files_decorator
+    def get_file_key(self, files):
+        """Get key of an annexed file.
+
+        Parameters:
+        -----------
+        files: list, str
+            file to look up
+
+        Returns:
+        --------
+        str
+            key used by git-annex for `path`
+        """
+
+        if len(files) > 1:
+            raise NotImplementedError("No handling of multiple files implemented yet for get_file_key()!")
+        path = files[0]
+
+        cmd_list = ['git', 'annex', 'lookupkey', path]
+        cmd_str = ' '.join(cmd_list)  # have a string for messages
+
+        try:
+            output = self.cmd_call_wrapper.run(cmd_list, return_output=True, cwd=self.path)
+        except CommandError, e:
+            if e.code == 1:
+                if not exists(p_join(self.path, path)):
+                    raise IOError(e.code, "File not found.", path)
+                elif path in self.get_indexed_files():
+                    # if we got here, the file is present and in git, but not in the annex
+                    raise FileInGitError(cmd=cmd_str, msg="File not in annex, but git: %s" % path,
+                                              filename=path)
+                else:
+                    raise FileNotInAnnexError(cmd=cmd_str, msg="File not in annex: %s" % path,
+                                               filename=path)
+            else:
+                # Not sure, whether or not this can actually happen
+                raise e
+
+        return output[0].split(linesep)[0]
+
+    @files_decorator
+    def file_has_content(self, files):
+        """ Check whether `files` are present with their content.
+
+        Note: Handling of multiple files not yet implemented!
+        TODO: Decide how to behave in case of multiple files, with some of them not present.
+
+        Parameters:
+        -----------
+        files: list
+            file(s) to check for being actually present.
+        """
+        # TODO: Also provide option to look for key instead of path
+        if len(files) > 1:
+            raise NotImplementedError("No handling of multiple files implemented yet for file_has_content()!")
+
+        cmd_list = ['git', 'annex', 'find']
+        cmd_list.extend(files)
```

Comment:
placing everything on a single line improves readability -- you can immediately catch what is going on:

```python
cmd_list = ['git', 'annex', 'find'] + files
```